File: expresslk-webportal/module/Api/src/Api/Client/Soap/Core/CancellationCriteria.php
<?php

namespace Api\Client\Soap\Core;

class CancellationCriteria
{

    /**
     * @var int $bookingId
     * @access public
     */
    public $bookingId = null;

    /**
     * @var CancellationCause $cause
     * @access public
     */
    public $cause = null;

    /**
     * @var boolean $chargeCancellationFee
     * @access public
     */
    public $chargeCancellationFee = null;

    /**
     * @var ItemCancellationCriteria[] $itemCriteria
     * @access public
     */
    public $itemCriteria = null;

    /**
     * @var string $remark
     * @access public
     */
    public $remark = null;

    /**
     * @param int $bookingId
     * @param CancellationCause $cause
     * @param boolean $chargeCancellationFee
     * @param string $remark
     * @access public
     */
    public function __construct($bookingId, $cause, $chargeCancellationFee, $remark)
    {
      $this->bookingId = $bookingId;
      $this->cause = $cause;
      $this->chargeCancellationFee = $chargeCancellationFee;
      $this->remark = $remark;
    }

}

File: expresslk-services/ExpressLK-core/src/main/java/lk/express/cnx/CancellationCriteria.java
package lk.express.cnx;

import java.util.List;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

@XmlRootElement
@XmlType(name = "CancellationCriteria", namespace = "http://cnx.express.lk")
public class CancellationCriteria {

	private int bookingId;
	private List<ItemCancellationCriteria> itemCriteria;
	private boolean chargeCancellationFee = true;
	private CancellationCause cause;

	private String remark;

	public int getBookingId() {
		return bookingId;
	}

	public void setBookingId(int bookingId) {
		this.bookingId = bookingId;
	}

	public List<ItemCancellationCriteria> getItemCriteria() {
		return itemCriteria;
	}

	public void setItemCriteria(List<ItemCancellationCriteria> itemCriteria) {
		this.itemCriteria = itemCriteria;
	}

	public boolean isChargeCancellationFee() {
		return chargeCancellationFee;
	}

	public void setChargeCancellationFee(boolean chargeCancellationFee) {
		this.chargeCancellationFee = chargeCancellationFee;
	}

	public CancellationCause getCause() {
		return cause;
	}

	public void setCause(CancellationCause cause) {
		this.cause = cause;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}
}
